docfx_project/ziti/quickstarts/network/hosted.md

```diff
@@ -2,35 +2,34 @@
 
 You can absolutely choose to host your [Ziti Network](xref:zitiOverview#overview-of-a-ziti-network) anywhere you like.
 It is not necessary for the server to be on the open internet. If it works better for you to deploy Ziti on your 
-own network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to 
-be able to communicate to the controller at least one edge router. If these components are all on your local network,
-that's perfectly fine. Just remember, the only components able to participate on this network will need to be able to
-contact the controller and at least one edge router.
+own private network, great, do that.  The only requirement to be aware of is that every piece of the a network will need to 
+be able to communicate to the Ziti Edge i.e. the control plane that is composed of the controller's edge-client REST API and 

```

Comment:
I still prefer the words that were there.  other than the awkward typo I just spotted: "every piece of the a network"...

"The only requirement to be aware of is that every piece of the a network will need to be able to communicate to the controller at least one edge router."